packages/components/src/product-icon/config.ts

```diff
@@ -68,7 +51,6 @@ export type SupportedSlugs =
 	| 'personal-bundle-monthly'
 	| 'value_bundle'
 	| 'value_bundle-2y'
-	| 'value_bundle-monthly'
```

Comment:
I think you'll need to keep this line to prevent a TS issue. Or remove it from the array associated to `wpcom-premium` in `iconToProductSlugMap`.

Reply:
Oops, I thought it was duplicated. Didn't notice the difference between `-` and `_` — thanks for catching that, @monsieur-z !

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/64144/commits/3f9de92486e31551bdf4f2baad69b9c8bbc774de